scale/Scaling_Deployments.md

```diff
@@ -0,0 +1,52 @@
+# Scaling Deployments
+
+This chapter depicts important information about scaling an existing WSO2 product deployment in a container platform.
+
+## Contents
+
+The following will be discussed in detail in the document.
+
+* [What is scaling?](#what-is-scaling?)
+* [When do we need to scale?](#when-do-we-need-to-scale?)
+* [Scaling metrics](#scaling-metrics)
+* [Scaling WSO2 product profiles](#scaling-wso2-product-profiles)
+* [Scaling strategies](#scaling-strategies)
+
+### What is scaling?
+
+In the context of a WSO2 product deployment, assuming that you have provided sufficient computing resources,
+scaling refers to increasing/decreasing the number of replicas of a given product profile.
+
+### When do we need to scale?
+
+The primary reason that may instigate scaling of a WSO2 product profile is the need to evenly accommodate the
+client request load.
```

Comment:
This is not the correct definition. Scaling should be talked when there is a need to scale beyond HA. The need is when the current TPS surpasses a certain threshold of the maximum TPS that can be handled by the currently available instances.